src/components/Footer.tsx

```diff
@@ -4,7 +4,11 @@ import ".././styles.css";
 // Footer with source code and project information that goes at the bottom of each page
 const Footer = (): ReactElement => {
   return (
+<<<<<<< HEAD
```

Comment:
Make sure to remove this `HEAD` and the `master` lines from the code, I think they're preventing it from deployment!